### PR TITLE
Create database list

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

It creates and adds a database list page.

#### Ticket
https://trello.com/c/Zr5tGzWa

#### Screenshots

Screenshot from 2021-02-11 10-47-14

Screenshot from 2021-03-02 04-49-11

### CODE REVIEW

**Review by MutegekiHenry — APPROVED** [review]

File: src/components/DatabaseList/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            </div>
+            <div className="ContentSection">
+              <div
+                className={
+                  isRetrieving
+                    ? "ResourcesTable LoadingResourcesTable"
+                    : "ResourcesTable"
+                }
+              >
+                <table>
```
Comment (colinwagaba) [inline src/components/DatabaseList/index.js:58]: i strongly discourage usage of tables, we shall need to rewrite this, and all the other `<table>` implementations

File: src/components/DatabaseList/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                  <thead>
+                    <tr>
+                      <th>Url</th>
```
Comment (colinwagaba) [inline src/components/DatabaseList/index.js:61]: also the fields changed, please review the design. thanks

**Review by colinwagaba — CHANGES_REQUESTED** [review]

File: src/components/DatabaseList/index.js (hunk 2 above)
Comment (neelxie) [reply src/components/DatabaseList/index.js]: @colinwagaba changes have been made kindly check it out

File: src/components/DatabaseList/index.js (hunk 1 above)
Comment (neelxie) [reply src/components/DatabaseList/index.js]: @colinwagaba kindly check out the new changes.

**Review by rajab256 — APPROVED** [review]

**Review by colinwagaba — APPROVED** [review]
we shall need to revisit some of the implementation details - table behaviour especially

**Pull request merged**